CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+Contributions to slide are warmly welcomed. Here's an overview of the code base of a slide and how to contribute!
```

Comment:
I think there are two big kinds of contributions -- filing issues and writing code. Can you describe how people can file issues and that we appreciate when they do? You can also mention the slide-bot for testing slide programs quickly.